lib/ember-test-helpers/test-module.js

```diff
@@ -230,6 +230,13 @@ export default Klass.extend({
     this.container = items.container;
     this.registry = items.registry;
 
+
+    var viewRegistry = this.container.lookup('-view-registry:main');
+    if (viewRegistry) {
+      var View = Ember.__loader.require('ember-views/views/view').default;
+      View.views = viewRegistry;
```

Comment:
Looking up the `EventDispatcher` from the container (as you changed above), should be sufficient to ensure that it internally has access to `-view-registry:main`, we should not need to set the internal `Ember.View.views`.